.formatter.exs

```diff
@@ -0,0 +1,4 @@
+[
+  inputs: ["lib/*.ex", "mix.exs", "test/*.exs"],
+  line_length: 120
+]
```

Comment:
ah not seen a formatter is this elixir specific? I know they are bringing out a built in one with 1.6

Reply:
Yup, it's for the built-in `mix format` command in 1.6!